### PR TITLE
DCS-1943 add record bodyscan link to post arrival summary page

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/views/pages/recentArrivals/recentArrivalsSummary.njk [hunk 1]
```diff
@@ -74,6 +74,10 @@
                     {{ summaryCard({
                         classes: "govuk-!-margin-bottom-6",
                         titleText: "X-ray body scan",
+                        actions: {items: [{ 
+                                    href: "/prisoners/" + arrival.prisonNumber + "/record-body-scan", 
+                                    text: "Record an X-ray body scan", 
+                                    attributes: { 'data-qa': 'record-body-scan' } }] if arrival.arrivalType == 'CURRENTLY_IN' else []},
```
Comment (louisenorris) [inline server/views/pages/recentArrivals/recentArrivalsSummary.njk:80]: Link should only show if person has been successfully booked into prison

File: server/views/components/summary-card.njk [hunk 2]
```diff
@@ -6,6 +6,11 @@
                 <h2 class="summary-card__title">
                     {{ params.titleHtml | safe if params.titleHtml else params.titleText }}
                 </h2>
+        {% if params.actions.items.length %}
+            <div class="app-summary-card__actions">
```
Comment (louisenorris) [inline server/views/components/summary-card.njk:10]: link should only show if person has been successfully booked into prison

File: server/views/components/summary-card.njk [hunk 3]
```diff
@@ -6,6 +6,11 @@
                 <h2 class="summary-card__title">
                     {{ params.titleHtml | safe if params.titleHtml else params.titleText }}
                 </h2>
+        {% if params.actions.items.length %}
```
Comment (louisenorris) [inline server/views/components/summary-card.njk:9]: link should only show if person has been successfully booked into prison

**Review by GurnankCheema — APPROVED** [review]
LGTM

File: server/views/components/summary-card.njk (hunk 3 above)
Comment (andrewrlee) [reply server/views/components/summary-card.njk:9]: should this loop round each action?

Comment (louisenorris) [reply server/views/components/summary-card.njk:9]: Opted for null check of `action.items` object as will only ever be one link within the summary card header. Refactored in [separate PR](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/pull/378).

**Pull request merged**